daal4py/sklearn/linear_model/_logistic_path_0_22.py

```diff
@@ -1012,29 +1009,176 @@ def daal4py_predict(self, X, resultsToEvaluate):
     raise ValueError('resultsToEvaluate must be in [computeClassLabels, \
                      computeClassProbabilities, computeClassLogProbabilities]')
 
+def daal4py_fit(self, X, y, sample_weight=None):
+    """Fit the model according to the given training data.
+
+    Parameters
+    ----------
+    X : {array-like, sparse matrix} of shape (n_samples, n_features)
+        Training vector, where n_samples is the number of samples and
+        n_features is the number of features.
+
+    y : array-like of shape (n_samples,)
+        Target vector relative to X.
+
+    sample_weight : array-like of shape (n_samples,) default=None
+        Array of weights that are assigned to individual samples.
+        If not provided, then each sample is given unit weight.
+
+    Returns
+    -------
+    self : object
+    """
+    solver = _check_solver(self.solver, self.penalty, self.dual)
+    daal_ready = solver in ['lbfgs', 'newton-cg'] and not sparse.issparse(X)
+    daal_ready = daal_ready and sample_weight is None and self.class_weight is None
+
+    if daal_ready:
+        logging.info("sklearn.linear_model.LogisticRegression.fit: " + get_patch_message("daal"))
+    else:
+        logging.info("sklearn.linear_model.LogisticRegression.fit: " + get_patch_message("sklearn"))
+
```

Comment:
I think we are unnecessarily copying so much code. Leave code branches that are only needed for call of daal4py